platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -2692,6 +2828,7 @@ namespace platoon_strategic_ihp
         // UCLA: assign a new plan type
```

Comment:
10 lines above here target_join_index is calculated by asking PlatoonManager about the index of platoon members. Problem is pm_ represents the hosts current platoon (as a solo vehicle), and not the platoon it desires to join.  So this info is meaningless.
I think what we need is for platoon_strategic_ihp to be able to look at INFO message from the neighbor platoon leader and from its leader DTD, size and length figure out the closest index.  As I have suspected for some time, the INFO message probably really needs to include something about each member (e.g. list of IDs in order); then a potential joiner can monitor the STATUS messages coming from each member to get their exact locations.  That would be a major change, so is worth discussing; we can probably come up with a better approach after some careful thought.